proof/access-control/CNode_AC.thy

```diff
@@ -1178,7 +1165,7 @@ lemma cap_swap_pas_refined[wp]:
      by (erule subsetD;
          force simp: is_transferable_weak_derived intro!: sbta_cdt_transferable auth_graph_map_memI)+
          apply (blast intro: state_bits_to_policy.intros auth_graph_map_memI)
-   by fastforce+
+   done
```

Comment:
Not directly related to this PR but I think that the indentation for the blast is off, probably due to the auto-indenter getting confused by the subgoal.